docs/workflows/lambda.md

```diff
@@ -6,123 +6,64 @@ hide_title: true
 
 # Develop Lambda Functions
 
-## Develop a new Lambda
+## Develop a new Cumulus Lambda
 
-To develop a new Lambda from a sample, create a new folder in `cumulus/tasks/` and run `npm init`:
+AWS provides great getting started guide for building Lambdas in the [developer guide](https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html).
 
-```bash
-  $ cd ../cumulus/tasks
-  $ mkdir new-lambda
-  $ cd new-lambda
-  $ npm init
-```
-
-Or copy an existing Lambda function to customize:
+Cumulus currently supports the following environments for Cumulus Message Adapter enabled functions:
 
-```bash
-  $ cd ../cumulus/tasks
-  $ cp discover-pdrs new-lambda
-```
+* [Node.js 8.10](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
+* [Java 8](https://docs.aws.amazon.com/lambda/latest/dg/java-programming-model.html)
+* [Python 2.7, 3.6](https://docs.aws.amazon.com/lambda/latest/dg/python-programming-model.html)
 
-Modify package.json:
+Addtionally you may chose to include any of the other languages AWS [supports](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) as a resource with reduced feature support.
 
-* name
-* version
-* description
-* test script
-* dependencies (NOT devDependencies)
+## Deploy a Lambda
 
-## Build a Lambda
+### Node.js Lambda
 
-To build Node.js Lambda functions, use webpack to pack the Lambda code into a single `.js` file with dependencies:
+For a new Node.js Lambda, create a new function and add an `aws_lambda_function` resource to your Cumulus deployment (for examples, see the example in source [example/lambdas.tf](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/lambdas.tf) and [ingest/lambda-functions.tf](https://github.com/nasa/cumulus/blob/master/tf-modules/ingest/lambda-functions.tf)) as either a new `.tf` file, or added to an existing `.tf` file:
 
-```bash
-  $ npm run build
-```
+```tcl
+resource "aws_lambda_function" "function" {
+  function_name    = "${var.prefix}-function"
+  filename         = "/path/to/zip/lambda.zip"
+  source_code_hash = filebase64sha256("/path/to/zip/lambda.zip/lambda.zip")
```

Comment:
extra `lambda.zip` at the end?

Reply:
Hmm, that's concerning, I'm pretty sure I block-copied that from elsewhere.   In either case: 781fe151f